Cargo.toml

```diff
@@ -16,6 +16,6 @@ build = ["nj-build"]
 
 [dependencies]
 nj-sys = { path = "nj-sys", version = "1.0.0", optional = true }
-nj-core = { path = "nj-core", version = "2.1.0", optional = true  }
-nj-build = { path = "nj-build", version = "0.2.0", optional = true }
-nj-derive = { path = "nj-derive", version = "2.0.0", optional = true}
+nj-core = { path = "nj-core", version = "2.1.1", optional = true  }
```

Comment:
I bumped these version in the main Cargo.toml file to be equal to their crate versions